src/interfaces/wasm_bindgen/tests.rs

```diff
@@ -1,3 +1,12 @@
+use abe_policy::{AccessPolicy, Attribute, Policy, PolicyAxis};
```

Comment:
Are you using the Rust `nightly` toolchain?

Reply:
yes, I confess :/